🌐 Add Portuguese translation for `docs/pt/docs/advanced/settings.md`

Joao-Pedro-P-Holanda opened the pull request:
Adding portuguese translation for `docs/pt/docs/advanced/settings.md`

Joao-Pedro-P-Holanda committed:
🌐 Add Portuguese translation for docs/pt/docs/advanced/settings.md

pre-commit-ci[bot] committed:
🎨 [pre-commit.ci] Auto format from pre-commit.com hooks

tiangolo commented:
📝 Docs preview for commit e2447763ef718c836b0919532a74a1ed094fb8dd at: https://dbae242b.fastapitiangolo.pages.dev

ceb10n commented on docs/pt/docs/advanced/settings.md:
```diff
@@ -0,0 +1,485 @@
+
+### Configurações e testes
+
+Dessa forma utilizar um objeto settings diferente durante os testes torna-se fácil ao criar uma sobrescrita de dependência em `get_settings`:
```
Eu traduziria essa frase como:

> Then it would be very easy to provide a different settings object during testing by creating a dependency override  for get_settings:

Então seria muito fácil fornecer uma configuração diferente durante a execução dos testes sobrescrevendo a dependência de `get_settings`:

ceb10n commented on docs/pt/docs/advanced/settings.md:
```diff
@@ -0,0 +1,485 @@
+
+`@lru_cache` modifica a função decorada para retornar o mesmo valor que foi retornado na primeira vez, em vez de calculá-lo novamente, executando o código da função toda vez.
+
+Assim, a função abaixo do decorador é executada uma única vez para cada combinação dos argumentos passados. E os valores retornados para cada combinação de argumentos são reutilizados de novo e de novo para cada chamada da função com a mesma combinação de argumentos.
```
Eu trocaria o de _novo e de novo_ daqui:

> combinação de argumentos são reutilizados de novo e de novo para cada chamada

Para:

combinação de argumentos são sempre reutilizados para cada chamada...

ceb10n reviewed (approved):
Very good translation, congrats! ;)

I added some suggestions, but it's a good translation whether you accept the suggestions or not.

Thank you

Joao-Pedro-P-Holanda commented:
@ceb10n thanks for the review, i would like to address these suggestions in a new commit before merging.

Joao-Pedro-P-Holanda committed:
chore: addressing suggested changes for 1st review

changing "reutilizados de novo e de novo para cada chamada da função" in line 420 to "sempre reutilizados para cada nova chamada da função"

restructuring phrase in line 308, and using the sentence "sobrescrevendo a dependência" instead of "sobrescrita de dependência"

tiangolo commented:
📝 Docs preview for commit b808dfc8616163306a17886405a5f424f60b28c4 at: https://3bc1dbff.fastapitiangolo.pages.dev

alejsdev commented:
Cool! Thanks to both of you for the help :nerd_face:  @Joao-Pedro-P-Holanda @ceb10n
